src/components/RichTextEditor/utils/parseRawValue.ts

```diff
@@ -26,7 +26,10 @@ export const parseRawValue = ({ editorId = 'parseRawValue', raw }: ParseRawValue
     try {
         parsedValue = JSON.parse(raw);
     } catch {
-        const editor = createPlateEditor({ id: `${editorId}_parseRawValue`, plugins: getEditorConfig() });
+        const editor = createPlateEditor({
+            id: `${editorId}_parseRawValue`,
+            plugins: createPlatePlugins(defaultPlugins),
```

Comment:
Maybe we should be able to pass in an editor itself rather than an editor id. Not sure 100% on the internals of Udecode/plate but it seems like we could parse a raw value incorrectly if we have a different selection of plugins to what is available in the RTE where this function is used, no? But for external use we can create a default editor inside the function

Reply:
This needs to have it, because it creates two different editors. We tried to pass it but at guideline blocks went in infinite loop. At the same moment you use two plates and one of them is just to parse the passed value.

Reply:
Ah ok, got it